data/test_domains/duplicate_entities.yml

```diff
@@ -8,8 +8,12 @@ intents:
 slots:
   cuisine:
     type: text
+    mappings:
+    - type: from_text
   location:
     type: text
+    mappings:
+    - type: from_text
```

Comment:
Yup, in this case, that's what will happen. However, because this particular domain didn't have too many details, I decided to use this mapping type since it seemed the easier route (this mapping type doesn't require other keys than `type`). Happy to change this to `custom`?

Reply:
I reckon if it hasn't come from a form we should have it as `custom` right?